rdt/transformers/base.py

```diff
@@ -145,16 +145,17 @@ def _get_columns_data(data, columns):
 
     @staticmethod
     def _set_columns_data(data, columns_data, columns):
```

Comment:
minor: can we rename this to `add_columns_to_data` and change the parameter names to `data, columns, column_names`?